tests/gis_tests/geoapp/test_indexes.py

```diff
@@ -2,12 +2,12 @@
 from django.db.models import Index
 from django.test import TransactionTestCase
 
-from .models import City
+from .models import City, SchemaCity
 
 
 class SchemaIndexesTests(TransactionTestCase):
     available_apps = []
-    models = [City]
+    models = [City, SchemaCity]
```

Comment:
Only works for postgis, will need another solution for this.

Reply:
I would create an inline model, see e.g. [test_namespaced_db_table_foreign_key_reference](https://github.com/django/django/blob/65a9d0013d202447dd76a9cb3c939aa5c9d23da3/tests/schema/tests.py#L3784-L3806).

Reply:
Perfect - thanks :+1: